include/dem/data_containers.h

```diff
@@ -0,0 +1,95 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 - 2022 by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+
+ *
+ */
+
+#include <deal.II/base/tensor.h>
+
+#include <deal.II/distributed/tria.h>
+
+#include <deal.II/particles/particle_iterator.h>
+
+#include <boost/range/adaptor/map.hpp>
+
+#include <iostream>
+#include <vector>
+
+#ifndef data_containers_h
+#  define data_containers_h
```

Comment:
```suggestion
#ifndef lethe_data_containers_h
#  define lethe_data_containers_h
```

This file could also be moved to core and renamed with a better file name. It would fit better this way.

Reply:
Moved to core.